scipy/special/_generate_pyx.py

```diff
@@ -968,7 +971,7 @@ def _get_python_wrap(self):
         for (outtype, _), outvar in zip(self.outtypes, self.outvars):
             line = "cdef {} {}".format(outtype, outvar)
             body.append(tab + line)
-        addr_outvars = map(lambda x: "&{}".format(x), self.outvars)
+        addr_outvars = ["&{}".format(x) for x in self.outvars]
```

Comment:
Another f string? `addr_outvars = [f"&{x}" for x in self.outvars]`

Reply:
done